src/System.Text.Json/src/System/Text/Json/Node/JsonArray.cs

```diff
@@ -87,14 +87,25 @@ public partial class JsonArray : JsonNode, IList<JsonNode>, IReadOnlyList<JsonNo
         [CLSCompliant(false)]
         public bool Contains(ulong value) { throw null; }
 
-        public int Count => throw new NotImplementedException();
-        public bool IsReadOnly => throw new NotImplementedException();
+        public int Count => throw null;
+        public bool IsReadOnly => throw null;
 
         public int IndexOf(JsonNode item) { throw null; }
+        public int LastIndexOf(JsonNode item) { throw null; }
         public void RemoveAt(int index) { throw null; }
         public void Clear() { throw null; }
         public bool Remove(JsonNode item) { throw null; }
 
+        /// <summary>
+        /// First by types of nodes (JsonBoolean, JsonNumber, JsonString, JsonArray, JsonObject
```

Comment:
Did we get this specific order from somewhere (like a spec or something) or is this just the one that we picked?

Reply:
Just my suggestion.

Reply:
(Found this one useful in: `TestSortingMultiTypedJsonArray`)